lib/Sema/CSStep.cpp

```diff
@@ -987,10 +1007,11 @@ StepResult ConjunctionStep::resume(bool prevFailed) {
   return take(prevFailed);
 }
 
-void ConjunctionStep::restoreOuterState() const {
+void ConjunctionStep::restoreOuterState(const Score solutionScore) const {
```

Comment:
Would ref make sense in this case?
```suggestion
void ConjunctionStep::restoreOuterState(const Score &solutionScore) const {
```

Reply:
I go back and forth on frankly, since Score is statically sized and pretty small I don’t think there is going to much difference in performance.

Reply:
Let's make it a `const Score &solutionScore` if we want to go this way...